geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/CreateAlterDestroyRegionCommandsDUnitTest.java

```diff
@@ -1091,6 +1086,126 @@ public void testDestroyRegionWithSharedConfig() {
     });
   }
 
+
+  final String PR_STRING = " package com.cadrdunit;"
+      + " public class TestPartitionResolver implements org.apache.geode.cache.PartitionResolver { "
+      + "   @Override" + "   public void close() {" + "   }" + "   @Override"
+      + "   public Object getRoutingObject(org.apache.geode.cache.EntryOperation opDetails) { "
+      + "    return null; " + "   }" + "   @Override" + "   public String getName() { "
+      + "    return \"TestPartitionResolver\";" + "   }" + " }";
+
+  /**
+   * Test Description 1. Deploy a JAR with Custom Partition Resolver 2. Create Region with Partition
+   * Resolver 3. Region should get created with no Errors 4. Verify Region Partition Attributes for
+   * Partition Resolver
+   * 
+   * @throws IOException
+   */
+  @Test
+  public void testCreateRegionWithPartitionResolver() throws IOException {
+    setUpJmxManagerOnVm0ThenConnect(null);
+    VM vm = Host.getHost(0).getVM(1);
+    // Create a cache in vm 1
+    vm.invoke(() -> {
+      assertNotNull(getCache());
+    });
+
+    ClassBuilder classBuilder = new ClassBuilder();
+    // classBuilder.addToClassPath(".");
+    final File prJarFile = new File(new File(".").getAbsolutePath(), "myPartitionResolver.jar");
```

Comment:
You should start using TemporaryFolder for stuff like this so the test cleans up better after it runs.

```
  @Rule
  public TemporaryFolder temporaryFolder = new SerializableTemporaryFolder();
...
    File prJarFile = new File(temporaryFolder.getRoot(), "myPartitionResolver.jar");

```
PS: SerializableTemporaryFolder is a a DUnit friendly version in geode-junit.